src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -59,23 +60,25 @@ export default function ConfirmedAppointmentListItem({
       <h3 className="vaos-appts__date-time vads-u-font-size--h3 vads-u-margin-x--0">
         {getAppointmentDateTime(appointment)}
       </h3>
-      <div className="vads-u-margin-top--2">
-        {canceled ? (
-          <i aria-hidden="true" className="fas fa-exclamation-circle" />
-        ) : (
-          <i aria-hidden="true" className="fas fa-check-circle" />
-        )}
-        <span
-          id={`card-${index}-state`}
-          className="vads-u-font-weight--bold vads-u-margin-left--1 vads-u-display--inline-block"
-        >
-          {canceled ? 'Canceled' : 'Confirmed'}
-        </span>
-      </div>
+      {(!isPastAppointment || canceled) && (
+        <div className="vads-u-margin-top--2 vads-u-margin-bottom--2">
+          {canceled ? (
+            <i aria-hidden="true" className="fas fa-exclamation-circle" />
+          ) : (
+            <i aria-hidden="true" className="fas fa-check-circle" />
+          )}
+          <span
+            id={`card-${index}-state`}
+            className="vads-u-font-weight--bold vads-u-margin-left--1 vads-u-display--inline-block"
+          >
+            {canceled ? 'Canceled' : 'Confirmed'}
+          </span>
+        </div>
+      )}
 
       <div className="vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
-        <div className="vads-u-flex--1 vads-u-margin-top--2 vads-u-margin-right--1 vaos-u-word-break--break-word">
-          {isVideoVisit(appointment) ? (
+        <div className="vads-u-flex--1 vads-u-margin-bottom--2 vads-u-margin-right--1 vaos-u-word-break--break-word">
+          {isVideoVisit(appointment) && !isPastAppointment ? (
```

Comment:
What does this look like now? It seems like we're showing the location info, which might be wrong?

Reply:
Hey @jbalboni, it just says "Video conference" so I think we should be okay assuming you are asking about a past appointment.  See screenshot below:

<img width="650" alt="Screen Shot 2020-03-31 at 10 55 35 AM" src="https://user-images.githubusercontent.com/786704/78059317-7a956880-733e-11ea-926f-c2fa7d2666e8.png">

Reply:
Cool, that seems fine